lib/platform/gitlab/index.ts

```diff
@@ -397,11 +397,8 @@ async function fetchPrList(): Promise<Pr[]> {
   const searchParams = {
     per_page: '100',
   } as any;
-  // istanbul ignore if
-  if (config.ignorePrAuthor) {
-    // https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests
-    // default: `scope=created_by_me`
-    searchParams.scope = 'all';
+  if (!config.ignorePrAuthor) {
```

Comment:
```suggestion
  if (authorId && !config.ignorePrAuthor) {
```

Reply:
This will likely remove the current snapshot changes, but also might require a new test to get coverage.

Reply:
Thanks for the suggestion! I've also coerced it as a string for strictness and added a test to show that we can still fetch MRs with `ignorePrAuthor` set.